Add accumulator page (#22)
ethereum/portal-website: pages/concepts/hash-accumulators.mdx

```

For post-merge blocks, there is an [equivalent accumulator](https://eth2book.info/capella/annotated-spec/#historicalsummary) already built into the main Ethereum chain, with the hashes being available in Beacon blocks.
This means portal clients do not have to implement their own accumulator for post-merge blocks, they can simply query Beacon data.

### Proofs

Proofs are used to verify some data against the accumulator. 
A [Merkle proof](https://pangea.cloud/docs/audit/merkle-trees) is generated using the hash of the block being verified as a leaf node and the root hash of the relevant epoch accumulator as the root.
The root hash of the epoch accumulator is the 'digest' and it is found in the master accumulator.
The proof asserts that a given hash is part of the accumulator. The accumulator is known to uniquely include canonical Ethereum data. 
Therefore, the proof demonstrates that a given block is canonical. Fake blocks and uncle blocks cannot produce valid proofs.
Proofs can be provided alongside any historical block data on the Portal Network, meaning that clients can instantly see whether the data is canonical or not at the moment it is received.

## Implementation

> The Portal Network specification does not define precisely how Portal clients acquire a master version of the accumulator to check new data against. 

Reviewer: ```suggestion
The Portal Network specification intentionally does not dictate how Portal clients acquire a master version of the accumulator to check new data against.
```

> It is expected that clients will likely maintain a 'frozen' copy of the pre-merge accumulator as this data can never change or grow and can be handled as a static dataset.

Reviewer: ```suggestion
Most clients will likely maintain a 'frozen' copy of the pre-merge accumulator as this data can never change or grow and can be handled as a static dataset.
```

The pre-merge accumulator only has to be built once, so it can be shipped with client code. Anyone can rebuild the accumulator to verify those hardcoded into clients using an Ethereum node. 

> For post-merge data, clients can make RPC calls to an Ethereum node to obtain the relevant hashes. 
> When the Portal Beacon Network is fully implemented, portal clients will be able to access this data without relying on access to an Ethereum node.

Reviewer: ```suggestion
For post-merge data, clients will be able to retrieve proofs from the Portal Beacon Network which is currently under active development.
```
